Point.java

```diff
@@ -0,0 +1,28 @@
+package sample;
+
+public class Point {
+    private int x_coordinate;
+    private int y_coordinate;
+
+
+    public Point(){
```

Comment:
Let's add a proper constructor that sets x/y coordinates directly, no need for setters.